site/src/sphinx/advanced-logging.rst

```diff
@@ -134,3 +139,24 @@ as the 3rd component of the ``<export />`` element in the XML configuration:
       </appender>
       ...
     </configuration>
+
+Customize MDC key
+-----------------
+You can override the pre-defined the MDC key by adding an alias in front of it.
```

Comment:
- the pre-defined the MDC key -> the pre-defined MDC key
- by prepending an alias and a equals sign (=) to it.
- How about making 'For example,...` appear before 'Note that ...'?